src/ext-test/java/org/opentripplanner/ext/fares/impl/GtfsFaresV2ServiceTest.java

```diff
@@ -37,82 +43,118 @@ class GtfsFaresV2ServiceTest implements PlanTestConstants {
   String expressNetwork = "express";
   String localNetwork = "local";
 
-  FareProduct single = new FareProduct(
-    new FeedScopedId(FEED_ID, "single"),
-    "Single one-way ticket",
-    Money.euros(1),
-    null,
-    null,
-    null
+  ArrayList<FareProduct> single = new ArrayList<>(
```

Comment:
Don't wrap `List.of` in `ArrayList` - it is already a `List`.

However, please keep those variables of type `FareProduct` and simply wrap them in `List.of` where needed.